### ISSUE TITLE
Document how to update translations

### ISSUE THREAD

**Issue by tordans** [maproulette/docs#37]
Do we have to update the strings like https://github.com/maproulette/docs/blob/4218de85035a5b2522e2860ea058f2e24cb288d2/src/translations/en-us.json#L31 manually inline or is there some UI?

Please add some docs that explain how to update the files.

**Comment by tordans** [comment]
Depending on the way those strings are created, it would be great if this could be refactored to use JS template literals, which would make the whole thing easier to read, get rid of the `\n` and improve the diff view. Diffs like https://github.com/maproulette/docs/commit/506de33d3e7c6dac7c80e5f0dc571f821fa65a13 are not really usefull…

**Comment by jschwarz2030** [comment]
I agree, updating the English translations does not have an ideal method.  For now I am using a JSON.parse web tool and converting it back to a string.  This is all for making the translation files compatible with transifex

For other languages, use transifex to update the translations.

> Depending on the way those strings are created, it would be great if this could be refactored to use JS template literals, which would make the whole thing easier to read, get rid of the `\n` and improve the diff view. Diffs like [506de33](https://github.com/maproulette/docs/commit/506de33d3e7c6dac7c80e5f0dc571f821fa65a13) are not really usefull…

Do you have a suggested approach for this that also allows it to work with transifex?

Thanks.

**Comment by jake-low** [comment]
Resolved by #52. Site content is once again stored in Markdown files, making editing the content much easier. Multilingual content is still supported - different versions are distinguished by filename extension, e.g. `some-page.en_US.md`, `some-page.pt_BR.md`, etc. New translations can be made by manually creating a new file with a new language in its extension, or using Transifex as before. See the [updated translation guidelines](https://learn.maproulette.org/en-US/documentation/translate-docs) for more details.
